tests/setup_cfg_fmt_test.py

```diff
@@ -62,16 +62,65 @@ def test_noop(tmpdir):
             '\n'
             '[metadata]\n'
             'version = 1.0\n'
-            'name = pkg\n',
+            'name = pkg\n'
+            '[options]\n'
+            'install_requires =\n'
+            '    req3\n'
+            '    req5 <= 2,!=1\n'
+            '    req6 ;python_version==2.7\n'
+            '    req7 ;os_version!=windows\n'
+            '    req1\n'
+            '    req2\n'
+            '    req9 ~= 7\n'
+            '    req10 === 8\n'
+            '    req8    ==    2\n'
+            '    req4 >= 1\n',
+
+            '[metadata]\n'
+            'name = pkg\n'
+            'version = 1.0\n'
+            '\n'
+            '[options]\n'
+            'install_requires =\n'
+            '    req1\n'
+            '    req10===8\n'
+            '    req2\n'
+            '    req3\n'
+            '    req4>=1\n'
+            '    req5!=1,<=2\n'
+            '    req8==2\n'
+            '    req9~=7\n'
+            '    req6;python_version==2.7\n'
+            '    req7;os_version!=windows\n'
+            '\n'
+            '[bdist_wheel]\n'
+            'universal = true\n',
+
+            id='orders fields and sections; normalizes requires',
```

Comment:
would prefer a separate test so it's clear what each change is doing -- this is testing 2 things at once now